.github/workflows/contribution_evaluation.yaml

```
@@ -0,0 +1,44 @@
+      - name: Run workload analysis script
+        id: analyze_workload
+        run: |
+          python scripts/workload_analyzation.py \
+            --owner ${{ github.repository_owner }} \
+            --repo ${{ github.event.repository.name }} \
+            --pr-number ${{ github.event.number }} \
+            --token ${{ secrets.GITHUB_TOKEN }} > workload_output.txt
+          cat workload_output.txt
+
+      - name: Post comment to PR
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)